diegodofe/MoreGlean
# feature/config-add-firebase

### What is this?

Resolves #4 

### Steps to QA

1. **Click** the _Get more gleans button_
2. **Ensure** you see ronaldo.
